tests/e2e/testcases/packageman/suite.go

```diff
@@ -144,7 +144,9 @@ var _ = ginkgo.Describe("[Package Management]", func() {
 
 		// check avago install
 		gomega.Expect(utils.CheckAvalancheGoExists(avagoVersion1)).Should(gomega.BeTrue())
-		gomega.Expect(utils.CheckAvalancheGoExists(avagoVersion2)).Should(gomega.BeFalse())
+		// FIXME: If we have to use twice the same version (because they are incompatible
```

Comment:
Can you add this to the E2E cleanup ticket?

Reply:
https://github.com/ava-labs/avalanche-cli/issues/420#issuecomment-1314287382